easybuild/easyconfigs/b/BAMM/BAMM-2.5.0-foss-2016b.eb

```diff
@@ -0,0 +1,39 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     GPL-v3.0
+#
+# Notes::
+##
+
+easyblock = 'CMakeMake'
+
+name = 'BAMM'
+version = '2.5.0'
+
+homepage = 'http://bamm-project.org/'
+description = """ BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates. 
+It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of 
+lineage diversification and trait evolution. """
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/macroevolution/bamm/archive']
+sources = ['v%(version)s.tar.gz']
+
+checksums = ['526eef85ef011780ee21fe65cbc10ecc62efe54044102ae40bdef49c2985b4f4', 'sha256']
```

Comment:
this is wrong, should be either:

```python
checksums = [('526eef85ef011780ee21fe65cbc10ecc62efe54044102ae40bdef49c2985b4f4', 'sha256')]
```

or 

```python
checksums = ['526eef85ef011780ee21fe65cbc10ecc62efe54044102ae40bdef49c2985b4f4']
```

I suggest the latter, where `eb` detects the checksum type automatically to be SHA256 based on the length, as we do elsewhere.